src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -337,10 +343,9 @@ public Flusher<ParquetChunkData> createFlusher() {
   private static class ParquetColumn {
```

Comment:
nit: move this class out to its' own file?

Reply:
Moved.